src/DynamoCore/Core/IDSDKManager.cs

```diff
@@ -240,11 +238,11 @@ private string IDSDK_GetToken()
 
         private bool Initialize()
         {
+            return false;
```

Comment:
hmm wouldn't this line make the following block irrelevant?

Reply:
Ah..yes was using this to test, will remove.